src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
This is not correct. Now your index is initialized when application starts and never changes. But you should generate new index every time method 'getRandomColor' is invoked
```suggestion
    private Random random = new Random();
```